docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
 sphinx
```

Comment:
We should completely delete this file

Reply:
https://github.com/deepchem/deepchem/pull/1102